nx.json

```diff
@@ -4,9 +4,10 @@
   },
   "tasksRunnerOptions": {
     "default": {
-      "runner": "nx/tasks-runners/default",
+      "runner": "@nrwl/nx-cloud",
       "options": {
-        "cacheableOperations": ["lint", "test", "types", "build"]
+        "cacheableOperations": ["lint", "test", "types", "build"],
+        "accessToken": "NGYxNTY3NTAtOWY5NC00MWVmLThiNWEtY2MwNjE4OWJjZTcxfHJlYWQ="
```

Comment:
This is the read-only token, e.g. you can only hit/miss remote cache but not upload the results. For the read-write token, you should create `nx-cloud.env` with the `NX_CLOUD_ACCESS_TOKEN` key and I can generate the token on demand. E.g. your execution results will be stored remotely and others can hit them if having the same execution hash.